### DISCUSSION TITLE
'./swiper-bundle.mjs': SyntaxError: Cannot use import statement outside a module (jest, react-testing-library)

### DISCUSSION THREAD

**Discussion by Darya7775** [nolimits4web/swiper#7206] [Q&A]
At testing to apeare erorr: 
    Details:

    C:\Users\hereAndNow\Рабочий стол\my_movies\node_modules\swiper\swiper-element-bundle.mjs:13
    import './swiper-bundle.mjs';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

Code: ` import Swiper from "swiper";
           import { register } from "swiper/element/bundle";
           import type { SwiperSlideProps, SwiperProps } from "swiper/react";`

**Comment by Frai** [comment]
I was facing the same issue and I solved it mocking the values:
```
jest.mock('swiper/modules', () => ({
    Navigation: jest.fn(),
    Pagination: jest.fn(),
    Virtual: jest.fn(),
}));

jest.mock('swiper/react', () => ({
    SwiperSlide: jest.fn(),
    useSwiper: jest.fn(),
    Swiper: jest.fn(),
}));

jest.mock('swiper/css', () => '');
jest.mock('swiper/css/pagination', () => '');
jest.mock('swiper/css/navigation', () => '');
```
